brendenwest/cascade
# Daily Rides Filter and Stack Implementation

## Description

Resolves issue #40 

This PR implements HomeStack which contains Home and Daily Rides screen. Navigation within the stack is supported by a button on home screen and stack header in the Featured Daily Rides screen. Adds Cascade Club logo to the Home page as an external asset.

Applies filter to the daily rides flatlist.
Current date is converted to a 'readable' format, then the date from API is sliced to fit that format, and when the dates match, the item renders on the screen.  Filtering supports one-digit and two-digit dates and unlimited number of rides for any current day.

Changed browsing method from Linking to InApp (thank you @zmagar for introducing)

## To Test
1. Run the app
2. Observe the home screen with the logo and Featured Daily Rides button. This is the initial stack landing screen.
3. Click on the 'Featured Daily Rides' button and observe only rides with the current date being rendered. When there are more than three rides, the list becomes scrollable. (Clicks on the rides turn the tile dark grey and take the user to external website links: pre-existing functionality)
3. Observe the back button in the top left corner. It navigates from the second stack screen Featured Daily Rides back to the Home screen.

**Note** for any style and additional feature suggestions, create stand-alone issues.
